web/src/app/modules/shared/components/abstract-view/abstract-view.component.ts

```diff
@@ -8,8 +8,8 @@ import {
 } from '@angular/core';
 import { View } from '../../models/content';
 
-@Directive()
-export abstract class AbstractViewComponent<T> // tslint:disable-line
+@Directive() // tslint:disable-line
+export abstract class AbstractViewComponent<T>
```

Comment:
Disable line should remain here for `ERROR: 12:23  directive-class-suffix  The name of the class AbstractViewComponent should end with the suffix Directive (https://angular.io/styleguide#style-02-03)`